.gitignore

```diff
@@ -33,3 +33,5 @@
 a.out
 *.out
 *.app
+
+build/
```

Comment:
The 🚫 means the the file does not end with a \n

Reply:
Agreed, will fix